homeassistant/components/remote/harmony.py

```diff
@@ -26,13 +26,15 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+ATTR_CHANNEL = 'channel'
 ATTR_CURRENT_ACTIVITY = 'current_activity'
 
 DEFAULT_PORT = 8088
 DEVICES = []
 CONF_DEVICE_CACHE = 'harmony_device_cache'
 
 SERVICE_SYNC = 'harmony_sync'
+SERVICE_CHANGE_CHANNEL = 'change_channel'
```

Comment:
Prefix the service name string with the name of the platform, separate with underscore. See the sync service.